app/components/NonDraggableElement.js

```diff
@@ -25,13 +24,11 @@ class NonDraggableElement extends Component {
 
   render() {
     const { text, id, size, source, type } = this.props;
+    const editable = false
```

Comment:
Why are you making a variable and passing it, instead of just passing the value on line 30?